_episodes/09-documentation.md

```diff
@@ -412,4 +412,45 @@ conda:
 Commit to these changes and push to your repository. You should now see your documentation strings.
 
 
-{% include links.md %}
\ No newline at end of file
+#### Automatically Generating Documentation Using Sphinx-AutoAPI
+Sphinx includes an alternative to autodoc/autosummary that will automatically generate an "API Reference" page based
+on the docstrings throughout the projet's source code upon compiling using `make html`.
+This is [Spinx-AutoAPI](https://github.com/readthedocs/sphinx-autoapi). To install Sphinx-AutoAPI, use `pip`:
+
+``pip install sphinx-autoapi``
+
+There are a few modifications necessary to use Sphinx-AutoAPI.  
+ - In the the conda-env file `devtools/conda-envs/test_env.yaml`, include `sphinx-autoapi` under
```

Comment:
I think this documentation dependency does not need to be added to the testing environment. Is there a reason to add it?

Reply:
I accidentally added this in when I was debugging. Indeed, this does not need to be in here. Deleted in latest commit.